drivers/LVHDoISCSISR.py

```diff
@@ -153,6 +153,7 @@ def load(self, sr_uuid):
                     self.session.xenapi.PBD.set_device_config(pbd, dconf)
             except:
                 util.logException("LVHDoISCSISR.load")
+                raise
```

Comment:
raising here has many unwanted effects.

Let's just raise if we did not manage to have any valid session (the reason why the next indexing fails).